### ISSUE TITLE
docker image ssh for sftp backend

### ISSUE THREAD

**Issue by ALUCAZEAU** [ngosang/restic-exporter#15]
Hi,
Thanks for your work !
You're docker image, does not contain ssh for use with sftp backend.
Exemple : 
`docker run -d --name=restic-exporter -e TZ=Europe/Paris -e RESTIC_REPO_URL=sftp:u3297xx@u3297xx.your-storagebox.de:/home/Nextcloud -e RESTIC_REPO_PASSWORD=yyyyyy -p 8001:8001 --restart unless-stopped ngosang/restic-exporter`
Error in journalctl : 
`exec: "ssh": executable file not found in $PATH  Exit code: 1`
Alexandre

**Comment by ngosang** [comment]
Included in c80b74c1ece917e4531721ee9df77f0f21d6f721
Release 1.2.2

**Comment by v4rakh** [comment]
@ALUCAZEAU Did you get it running? I always get the following:

```shell
Unable to collect metrics from Restic. Exception: Error executing restic snapshot command: Fatal: parsing repository location failed: invalid backend If the repository is in a local directory, you need to add a `local:` prefix  Exit code: 1
```
